app/models/user.rb

```diff
@@ -3,6 +3,9 @@ class User < ApplicationRecord
 
   TOKEN_EXPIRATION_TIME = 15.minutes
 
+  has_many :company_members, dependent: :destroy
+  has_many :articles, dependent: :nullify
```

Comment:
I see that `user_id` is required in the article model (`belongs_to :user`). But it seems the value can become empty if we delete the user.